packages/expo/src/winter/runtime.ts

```diff
@@ -0,0 +1,5 @@
+Object.defineProperty(globalThis, 'ExpoImportMetaRegistry', {
+  value: require('./ImportMetaRegistry').ImportMetaRegistry,
+  enumerable: true,
```

Comment:
Is this intended to be a public API? If not, I'm wondering if enumerable: false is better (and possibly a name like `__ExpoImportMetaRegistry`)